### PR TITLE
grafana-pd-dashboard: Update pd dashboard

### PR DESCRIPTION
**Pull request by shenli**
### What is changed, added or deleted? (Required)
Update pd dashboard according to https://github.com/pingcap/docs-cn/pull/3343

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3343
- Other reference link(s):

### CODE REVIEW

File: grafana-pd-dashboard.md [hunk 1]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
+- Current storage size: The current storage size of the cluster
+- Current storage usage: The total number of Regions without replicas
+- Normal stores: The count of healthy storage instances
+- Number of Regions: The total count of Regions of the cluster 
+- Abnormal stores: The count of unhealthy stores. The normal value is `0`. If the number is bigger than `0`, it means at least one instance is abnormal.
+- Region health: The health status of Regions indicated via count of unusual Regions including pending peers, down peers, extra peers, offline peers, missing peers, learner peers and incorrect namespaces. The number of pending peers should be less than `100`. The missing peers should not be persistently greater than `0`.
+- Current peer count: The current peer count of the cluster
+![PD Dashboard - Header](/media/pd-dashboard-header-v4.png)
+
 ## Key metrics description
```
Comment (lilin90) [inline grafana-pd-dashboard.md:29]:
```suggestion
## Key metrics description

```

File: grafana-pd-dashboard.md [hunk 2]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
+
+- PD scheduler config: The list of PD scheduler configurations
+- Cluster ID: The unique identifier of the cluster
+- Current TSO: The physical part of current allocated TSO
+- Current ID allocation: The maximun allocatable ID for new store/peer
+- Region label isolation level: The number of Regions in different label levels
+- Label distribution: The distribution status of the labels in the cluster
+![PD Dashboard - Cluster metrics](/media/pd-dashboard-cluster-v4.png)
+
+
```
Comment (lilin90) [inline grafana-pd-dashboard.md:40]:
```suggestion
```

File: grafana-pd-dashboard.md [hunk 3]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
 ## Key metrics description
+## Cluster
+
+- PD scheduler config: The list of PD scheduler configurations
+- Cluster ID: The unique identifier of the cluster
+- Current TSO: The physical part of current allocated TSO
+- Current ID allocation: The maximun allocatable ID for new store/peer
+- Region label isolation level: The number of Regions in different label levels
+- Label distribution: The distribution status of the labels in the cluster
+![PD Dashboard - Cluster metrics](/media/pd-dashboard-cluster-v4.png)
```
Comment (lilin90) [inline grafana-pd-dashboard.md:38]:
```suggestion

![PD Dashboard - Cluster metrics](/media/pd-dashboard-cluster-v4.png)
```

File: grafana-pd-dashboard.md [hunk 4]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
+- 99% Handle transactions duration: The transaction handling rate (P99)
+- 99% WAL fsync duration: The time consumed for writing WAL into the persistent storage (P99) | The value is less than `1s`
+- 99% Peer round trip time seconds: The network latency for etcd (P99) | The value is less than `1s`
+- etcd disk WAL fsync rate: The rate of writing WAL into the persistent storage
+- Raft term: The current term of Raft
+- Raft committed index: The last committed index of Raft
+- Raft applied index: The last applied index of Raft
+
+![PD Dashboard - etcd metrics](/media/pd-dashboard-etcd-v2.png)
+
```
Comment (lilin90) [inline grafana-pd-dashboard.md:134]:
```suggestion
```

**Comment by shenli** [comment]
@nolouch PTAL

File: grafana-pd-dashboard.md [hunk 5]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
+- Store Region size: The total Region size per TiKV instance
+- Store leader count: The leader count per TiKV instance
+- Store Region count: The Region count per TiKV instance
+
+![PD Dashboard - Balance metrics](/media/pd-dashboard-balance-v4.png)
+
+## Statistics - hot write
+
+- Hot Region's leader distribution: Leader distribution in hot write Regions. The total number of leader Regions in hot write on each TiKV instance
+- Total written bytes on hot leader Regions: The total written bytes by Leader regions in hot write on leader Regions for each TiKV instance
```
Comment (lilin90) [inline grafana-pd-dashboard.md:74]:
```suggestion
- Total written bytes on hot leader Regions: The total written bytes by Leader Regions in hot write for each TiKV instance
```

File: grafana-pd-dashboard.md [hunk 6]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
+- Completed commands rate: The rate per command type type at which gRPC commands are completed
+- 99% Completed commands duration: The rate per command type type at which gRPC commands are completed (P99)
+
+![PD Dashboard - gRPC metrics](/media/pd-dashboard-grpc-v2.png)
+
+## etcd
+
+- Handle transactions count: The rate at which etcd handles transactions
+- 99% Handle transactions duration: The transaction handling rate (P99)
+- 99% WAL fsync duration: The time consumed for writing WAL into the persistent storage (P99) | The value is less than `1s`
```
Comment (lilin90) [inline grafana-pd-dashboard.md:126]:
```suggestion
- 99% WAL fsync duration: The time consumed for writing WAL into the persistent storage. It is less than `1s` (P99)
```

File: grafana-pd-dashboard.md [hunk 7]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
-
-![PD Dashboard - Operator metrics](/media/pd-dashboard-operator-v2.png)
-
-### gRPC
 
-![PD Dashboard - gRPC metrics](/media/pd-dashboard-grpc-v2.png)
+## TiDB
 
-### etcd
+- PD Server TSO handle time and Client recv time: The duration from PD receive TSO request to PD client get the TSO respones
```
Comment (lilin90) [inline grafana-pd-dashboard.md:141]:
```suggestion
- PD Server TSO handle time and Client recv time: The duration between PD receiving the TSO request and the PD client getting the TSO response
```

File: grafana-pd-dashboard.md [hunk 8]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
+- Handle requests count: The count of TiDB requests
+- Handle requests duration: The time consumed for handling TiDB requests | It should be less than `100ms` (P99)
```
Comment (lilin90) [inline grafana-pd-dashboard.md:143]:
```suggestion
- Handle requests duration: The time consumed for handling TiDB requests. It should be less than `100ms` (P99)
```

File: grafana-pd-dashboard.md [hunk 9]
```diff
@@ -13,122 +13,151 @@ The Grafana dashboard is divided into a series of sub dashboards which include O
...
 
-![PD Dashboard - etcd metrics](/media/pd-dashboard-etcd-v2.png)
+![PD Dashboard - TiDB metrics](/media/pd-dashboard-tidb-v4.png)
 
-### TiDB
+## Heartbeat
 
-![PD Dashboard - TiDB metrics](/media/pd-dashboard-tidb-v2.png)
+- Heartbeat region event QPS: The QPS of handling heartbeat message, inlcuding updating cache and persisting data
```
Comment (lilin90) [inline grafana-pd-dashboard.md:149]:
```suggestion
- Heartbeat region event QPS: The QPS of handling heartbeat messages, including updating the cache and persisting data
```

**Pull request merged**